Week1/Section3/Exercise8/Source8.cpp

```diff
@@ -15,8 +15,8 @@ int main()
 
 	/* initialize random seed: */
 	srand(time(NULL));
-
-	for (int i = 0; i < sizeof(Lottery) / sizeof(int); i++)
+	int LotteryArraySize = sizeof(Lottery) / sizeof(int);
+	for (int i = 0 ; i < LotteryArraySize ; i++)
```

Comment:
I see you added a space here, but there should be no spaces before a semicolon.

Reply:
jajaj I thought you would say something for the missing space xd I delete it from everywhere ✌